src/components/PostList.tsx

```diff
@@ -24,17 +24,31 @@ interface PostListProps {
   feedType: FeedTypes;
 }
 
+const sortFeed = (feed: FeedItem[]): FeedItem[] => {
+  feed.sort(
+    (firstFeedItem: FeedItem, secondFeedItem: FeedItem) =>
+      secondFeedItem.blockNumber - firstFeedItem.blockNumber
```

Comment:
This is okay in Foodee since we are publishing a batch every time we make an announcement, but we will need to also use transaction and log index as well because blockNumber will be the same in such a case.